gui/wxpython/nviz/mapwindow.py

```diff
@@ -382,10 +382,7 @@ def OnEraseBackground(self, event):
 
     def OnSize(self, event):
         size = self.GetClientSize()
-        if CheckWxVersion(version=[2, 9]):
```

Comment:
We may as well just remove the check, it's too old version

Reply:
Without docs, I'm not sure what does the check return true for (to choose what branch). If you don't know I'll read later